docker/primer_schemes/ARTIC/SARS-CoV-2/V4/SARS-CoV-2.scheme.fasta

```diff
@@ -155,7 +155,7 @@ CAAACCTTTTGAGAGAGATATTTCAACTGA
 CACTGACACCACCAAAAGAACATG
 CTGAGTCTAACAAAAAGTTTCTGCCTT
 GGATTGACTAGCTACACTACGTGC
-ACCCATTGGTGCAGGTATAGC
+ACCCATTGGTGCAGGTATATGC
```

Comment:
The file to the left contains the sequences from the artic network github for V4.
 
The sequence to the left is incorrect and in fact it was fixed in V4-1 to what we have to the right.

Therefore, aside from this fix, the script introduced with this PR matches 100% against the primer sequences retrieved from the artic repo for V2, V3, V4 and V4-1. Here we add V1 simply because I did not extract the sequences for this version from the artic repo.
Therefore, I have strong reasons to believe that the procedure is correct and we have correct sequences for the other primers too.